amp.php

```diff
@@ -30,6 +30,23 @@ function _amp_print_php_version_admin_notice() {
 	return;
 }
 
+/**
+ * Print admin notice regarding DOM extension is not installed
+ *
+ * @since 0.7.2
```

Comment:
This should be changed to `1.1` if merging into `develop`. However, if you rebase your commits onto the `1.0` branch and change the PR to point to the `1.0` branch, then the version can be changed to `1.0` since this would be good to include sooner.